packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -0,0 +1,84 @@
+import { useCallback, useMemo, useState } from "react";
+import {
+  type Expression,
+  unwrapWidget,
+  useRegisterBindings,
+  type EnsembleAction,
+} from "@ensembleui/react-framework";
+import { Collapse } from "antd";
+import type { CollapseProps } from "antd";
+import { isArray, isString } from "lodash-es";
+import type { EnsembleWidgetProps } from "../shared/types";
+import { WidgetRegistry } from "../registry";
+import { EnsembleRuntime } from "../runtime";
+import { useEnsembleAction } from "../runtime/hooks/useEnsembleAction";
+
+interface CollapsibleItem {
+  key: Expression<string>;
+  label: Expression<string> | Record<string, unknown>;
+  children: Expression<string> | Record<string, unknown>;
+}
+
+export type CollapsibleProps = {
+  items: CollapsibleItem[];
+  expandIconPosition?: "start" | "end";
+  value: Expression<string>[];
+  onCollapse?: EnsembleAction;
+  isAccordion?: boolean;
+} & EnsembleWidgetProps;
+
+export const Collapsible: React.FC<CollapsibleProps> = (props) => {
+  const [activeValue, setActiveValue] = useState<string[]>(props.value);
```

Comment:
Does this represent whether it's collapsed or not?

Reply:
This collapsible widget takes an array of labels and children as input. Based on this array, it creates a list of collapsibles. If the activeValue state has any value, it indicates that one or multiple collapsibles from that array are collapsed.

Reply:
I see. We will need to collapse items individually and read their value. The invokable method that is registered should be a callback that takes the index and value to make it easier to use `(index: int, value: string) => void` over passing an entire array.